cccl

```diff
@@ -30,17 +30,10 @@ given by [OPTIONS] that Unix cc understands into parameters that cl understands.
 EOF
 }
 
-case $MACHTYPE in
-    *-msys)
-        slash="-"
-        ;;
-    *)
-        slash="/"
-        ;;
-esac
+slash="-"
```

Comment:
This was already not optimal before, but now this seems downright wrong to call a variable always containing dash `slash`. IMO it should be renamed to `optsep` or something like this.